verilog/tools/ls/autoexpand.cc

```diff
@@ -305,8 +308,10 @@ class AutoExpander {
   // Symbol table wrapper for the language server
   SymbolTableHandler *symbol_table_handler;
 
-  // String to add at the end of each generated line
+  // Indentation in various contexts
```

Comment:
Instead of the strings containing indentation spaces, you could consider using `verible::Spacer()`, (common/util/spacer.h). Up to you.